pkg/apb/types.go

```diff
@@ -53,7 +53,6 @@ type Context struct {
 // ExtractedCredentials - Credentials that are extracted from the pods
 type ExtractedCredentials struct {
 	Credentials map[string]interface{} `json:"credentials,omitempty"`
-	// might be more one day
```

Comment:
Ok to remove but it was still a true statement :) bind could return:
https://github.com/openservicebrokerapi/servicebroker/blob/master/spec.md#types-of-binding